apps/patient/src/views/Pharmacy.tsx

```diff
@@ -197,16 +198,43 @@ export const Pharmacy = () => {
           is24hr: enable24Hr,
           name: 'costco'
         });
-        if (topRankedPharmacy.length > 0) {
+        if (topRankedCostco.length > 0) {
           // add a logo to the only item in the array
-          topRankedPharmacy[0].logo = costcoLogo;
+          topRankedCostco[0].logo = costcoLogo;
+          topRankedPharmacies.push(topRankedCostco[0]);
         }
       }
     } catch {
       // no costcos found :(
       pharmaciesResult = [];
     }
 
+    // check if top ranked walgreens is in the area
+    try {
+      if (order.organization.name === 'Blueberry Pediatrics' && order.readyBy === 'Urgent') {
```

Comment:
I think you could release this before Monday, in the costco case in the same file I flag it in the orgSettings:

``` ts
const enableTopRankedCostco = !isDemo && orgSettings.topRankedCostco;
// ... 
if (enableTopRankedCostco && containsGLP) {
```

You would then updated the org setting on monday to released it to Blueberry.

I could be convinced of your approach for flagging a customer too, but it would be good if they were consistent.

Reply:
oh yeah, way better, I'll do that